lib/models/blueprint.js

```diff
@@ -894,6 +894,60 @@ Blueprint.prototype.addPackagesToProject = function(packages) {
 };
 
 /**
+  Used to remove a package from the projects `package.json`.
+
+  Generally, this would be done from the `afterInstall` hook, to
```

Comment:
It looks like this description is a copy-past of the addPackage? Could you fix that please.